src/WebJobs.Script/Utility.cs

```diff
@@ -497,6 +497,19 @@ internal static string GetWorkerRuntime(IEnumerable<FunctionMetadata> functions)
             return null;
         }
 
+        internal static bool IsSupported(FunctionMetadata functionMetadata, string workerRuntime)
```

Comment:
This needs a better name now that it is in this generic class.

Reply:
gave a really long name :)